seqr/urls.py

```diff
@@ -269,10 +271,15 @@
     'matchmaker/v1/match': external_api.mme_match_proxy,
     'matchmaker/v1/metrics': external_api.mme_metrics_proxy,
 
+    'create_project_from_workspace/submit/(?P<namespace>[^/]+)/(?P<name>[^/]+)': create_project_from_workspace,
 }
 
 urlpatterns = [url('^status', status_view)]
 
+# anvil workspace
+anvil_workspace_url = 'anvil_workspace/(?P<namespace>[^/]+)/(?P<name>[^/]+)'
```

Comment:
messaged in slack, but just to record it on the PR as well the AnVIL team requested we change the url to `'workspace/(?P<namespace>[^/]+)/(?P<name>[^/]+)'`

Reply:
It belongs to the backend. I updated it in the backend branch.